### DISCUSSION TITLE
Feature request: Subscription for OSS projects

### DISCUSSION THREAD

**Discussion by seito2** [community/community#9945] [Other Feature Feedback, Questions, & Ideas]
Now, many OSS projects are lack of funding. So you need to sponsor the package we are using. But there are so many repositories, and it's hard to decide on one. 

So if you can, I want subscription program for sponsoring OSS projects, like music subscription.

Thanks.

**Comment by jlord** [comment]
Hiya! Have you seen our [Sponsors Explore](https://github.com/sponsors/explore) page? It uses your project dependencies (personal and any org you belong to) to show you what dependencies you have that are sponsor-able. Would be great to hear your thoughts on how that helps achieve this. 

Additionally, how would you expect this subscription service might work? What types of projects and what budget? Thanks, this feedback is really helpful!
